moduleroot/.github/workflows/acceptance.yml.erb

```
@@ -0,0 +1,74 @@
+<%- if Dir[File.join(@metadata[:workdir], 'spec', 'acceptance', '**', '*_spec.rb')].any? -%>
+name: Acceptance tests
+
+on: pull_request
+
+jobs:
+  build_cache:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: "2.7"
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-2.7-gems-acceptance-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-2.7-gems-acceptance-
+      - name: Install dependencies
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle config without 'development test'
+          bundle install --jobs 4 --retry 3
```

Hi!
small question here. I'm not sure how many cores github exposes and if the following work:
```suggestion
          bundle install --jobs $(nproc) --retry 3
```

but at least locally this gives me a huge performance improvement while installing gems